osquery/sql/sqlite_util.cpp

```diff
@@ -160,6 +212,52 @@ SQLiteDBManager::~SQLiteDBManager() {
   }
 }
 
+QueryPlanner::QueryPlanner(const std::string& query, sqlite3* db) {
+  queryInternal("EXPLAIN QUERY PLAN " + query, plan_, db);
+  queryInternal("EXPLAIN " + query, program_, db);
+
+  for (const auto& row : plan_) {
+    auto details = osquery::split(row.at("detail"));
+    tables_.push_back(details[2]);
```

Comment:
If the tables are deduced here, we do not need to keep the `QUERY PLAN` results around as a member variable.